tests/planner_test/tpch.planner.sql

```diff
@@ -215,10 +227,11 @@ where
     and l_quantity < 24;
 
 /*
-Projection: [sum((l_discount * l_extendedprice))] (cost=7409.928)
-  Aggregate: [sum((l_discount * l_extendedprice))], groupby=[] (cost=7408.328)
-    Filter: ((l_shipdate >= 1994-01-01) and ((1995-01-01 > l_shipdate) and ((0.09 >= l_discount) and ((24 > l_quantity) and (l_discount >= 0.07))))) (cost=7210.72)
-      Scan: lineitem[l_quantity, l_extendedprice, l_discount, l_shipdate] (cost=4000)
+Projection: [sum((l_discount * l_extendedprice))] (cost=8163.5923)
+  Aggregate: [sum((l_discount * l_extendedprice))], groupby=[] (cost=8161.992)
+    Projection: [l_extendedprice, l_discount] (cost=7964.3843)
```

Comment:
From the execution point of view, I don't expect the new plan to be faster given this project is simply extracting two columns out of the data chunk.

Reply:
Sure. The projection here is unnecessary. (But also won't be a big impact since it's a simple extraction.) It should be eliminated when the cost function gets smart enough to find it unnecessary. 😇